spec/takeaway_menu_spec.rb

```diff
@@ -2,10 +2,10 @@
 
 describe TakeAwayMenu do
 
-  let(:subject) {described_class.new}
+  let(:subject) { described_class.new }
 
   it 'contains a readable menu' do
-      expect(subject.read["spring roll"]).to eq 0.99
+    expect(subject.read["spring roll"]).to eq 0.99
```

Comment:
You use the read method here.  Is this widely available in other programming languages?